worker-example/worker-example-container/pom.xml

```diff
@@ -71,12 +71,6 @@
             <groupId>com.github.workerframework</groupId>
             <artifactId>standard-worker-container</artifactId>
             <type>pom</type>
-            <exclusions>
-                <exclusion>
-                    <groupId>com.github.workerframework</groupId>
-                    <artifactId>worker-store-cs</artifactId>
-                </exclusion>
-            </exclusions>
         </dependency>
         <dependency>
             <groupId>com.github.workerframework</groupId>
```

Comment:
This worker-store-fs dependency should now be removed, since it is coming from `standard-worker-container`.